pkg/operation/botanist/component/networkpolicies/global.go

```diff
@@ -38,8 +38,8 @@ type GlobalValues struct {
 	// DenyAllTraffic states whether all traffic should be denied by default and must be explicitly allowed by dedicated
 	// network policy rules.
 	DenyAllTraffic bool
-	// NodeLocalIPVSAddress is the CIDR of the node-local IPVS address.
-	NodeLocalIPVSAddress *string
+	// IPVSAddress is the CIDR of the node-local IPVS address.
+	IPVSAddress *string
```

Comment:
Why was it renamed here as well? This is the `networkpolicies` component, i.e., being more explicit is helpful (now, just with `IPVSAddress` it's not clear whose address this is).

Reply:
It is also about the `node-local` IPVS address, shouldn't it be the same all over the code?

Reply:
Again, this is the `networkpolicies` package. Naming the attribute `IPVSAddress` doesn't tell anybody that it's about node-local-dns. Hence, please rename it again to `NodeLocalIPVSAddress` so that it's clear for consumers. In the `nodelocaldns` package, it's a different storage since the package name already provides context.